### DISCUSSION TITLE
Add the ability to preview releases

### DISCUSSION THREAD

**Discussion by James-Jackson-South** [community/community#6099] [Other Feature Feedback, Questions, & Ideas] [answered]
I've enabled the releases beta however attempting to draft and preview a new release doesn't present me with any information. 

Are there additional mandatory steps? i.e. releases.yml, if so it should be optional  falling back to to a default.

**Comment by James-Jackson-South** [comment, marked as answer]
Spotted the invisible button

https://github.com/github/feedback/discussions/6098#discussioncomment-1445394

**Reply by MylesBorins** [reply]
There is a preview button to view the output of the text and you can save a release as a draft. Is there additional functionality you are looking for?

**Reply by James-Jackson-South** [reply]
No. I just hadn't spotted the button - as linked.
